roles/elastic-stack/ansible-kibana/tasks/Debian.yml

```diff
@@ -10,12 +10,13 @@
 
 - name: Debian/Ubuntu | Add Elasticsearch GPG key
   apt_key:
-    url: "https://artifacts.elastic.co/GPG-KEY-elasticsearch"
+    url: "{{ elasticrepo.gpg }}"
+    id: "{{ elasticrepo.kid }}"
```

Comment:
Just a little tweak, I think `elasticrepo.key_id` is much more understable than `elasticrepo.kid`. Please consider changing it.

Thanks for your patience!

Reply:
done